libs/dh/metering-point/feature-overview/src/lib/primary-master-data/dh-metering-point-primary-master-data.component.spec.ts

```diff
@@ -0,0 +1,326 @@
+/**
+ * @license
+ * Copyright 2021 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { render, within } from '@testing-library/angular';
+
+import { en as enTranslations } from '@energinet-datahub/dh/globalization/assets-localization';
+import { getTranslocoTestingModule } from '@energinet-datahub/dh/shared/test-util-i18n';
+import {
+  getByTerm,
+  getDefinitonByTerm,
+  queryByTerm,
+  runOnPushChangeDetection,
+} from '@energinet-datahub/dh/shared/test-util-metering-point';
+
+import {
+  DhMeteringPointPrimaryMasterDataComponent,
+  DhMeteringPointPrimaryMasterDataScam,
+  PrimaryMasterData,
+} from './dh-metering-point-primary-master-data.component';
+import { emDash } from '../identity/em-dash';
+
+describe(DhMeteringPointPrimaryMasterDataComponent.name, () => {
+  const fallbackValue = emDash;
+  const testData: PrimaryMasterData = {
+    meterId: '000',
+    supplyStart: '2020-01-01',
```

Comment:
suggestion: Let's use a full datetime value here (with UTC). The test will break when the date pipe is used but that will be fixed as part of that user story.